templates/wizard.php

```diff
@@ -63,7 +63,7 @@
 	and <a href="http://owncloud.org/promote"
 		target="_blank">spread the word</a>!')); ?>
 <?php else: ?>
-© 2014 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
+© 2015 <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
```

Comment:
@karlitschek shouldn't that be a period like 2012-2015 - just from a legal pov? THX